### PR TITLE
kubernetes-helm 2.3.1

### PR DESCRIPTION
**Pull request by cblecker**
Created with `brew bump-formula-pr`.

---
fixes #12263

### CODE REVIEW

**Comment by cblecker** [comment]
In addition to version bump, I've added an install for the man pages, and a test to ensure the issue in #12263 doesn't crop up again.

cc: @ilovezfs

File: Formula/kubernetes-helm.rb [hunk 1]
```diff
@@ -45,5 +48,6 @@ def install
 
     version_output = shell_output("#{bin}/helm version --client 2>&1")
     assert_match "GitTreeState:\"clean\"", version_output
+    assert_match stable.instance_variable_get(:@resource).instance_variable_get(:@specs)[:revision], version_output if build.stable?
```
Comment (DomT4) [inline Formula/kubernetes-helm.rb]: As far as I know nobody fixed this, and if that's the case, `build.xyz` doesn't work in `test do`. You'd need to use `if.stable?`.

Comment (cblecker) [inline Formula/kubernetes-helm.rb]: @DomT4:`if.stable?` gives a ruby syntax error. should it be `if stable?` ?

Comment (cblecker) [inline Formula/kubernetes-helm.rb]: Actually, I just tested this locally and `if build.stable?` seems to do what's expected:
- If I install stable and test, it passes.
- If I install stable, modify the revision in the formula, and then test, it fails with expected output.
- If I install HEAD and test, it passes.
- If I install HEAD, but remove `if build.stable?` and test, it fails with expected output.

**Comment by DomT4** [comment]
Yeah, I meant if stable?, sorry, rather late here. The build part used to be effectively nil outside of the prepare/install stage. Maybe someone finally fixed that though.

File: Formula/kubernetes-helm.rb (hunk 1 above)
Comment (DomT4) [inline Formula/kubernetes-helm.rb]: Yeah, I meant `if stable?`, sorry. The build part used to be effectively nil outside of the prepare/install stage. Maybe someone finally fixed that though.

_(Apologies for the repost; when I replied by email GitHub didn't stick the comment in the right discussion thread)_

Comment (ilovezfs) [inline Formula/kubernetes-helm.rb]: Yeah both `build.stable?` and `stable?` work in the test block currently.

**Comment by ilovezfs** [comment]
Thanks @cblecker

**Pull request closed without merging**